utils/__tests__/conversions.test.ts

```diff
@@ -45,6 +45,7 @@ import { readableReport } from "italia-ts-commons/lib/reporters";
 import { FiscalCode, NonEmptyString } from "italia-ts-commons/lib/strings";
 import { BonusVacanzaBase as ApiBonusVacanzaBase } from "../../generated/ade/BonusVacanzaBase";
 import { BonusActivationStatusEnum as ApiBonusActivationStatusEnum } from "../../generated/definitions/BonusActivationStatus";
+import { BonusCode } from "../../generated/definitions/BonusCode";
```

Comment:
`definitions` are for api, should we consider domain types here?

```suggestion
import { BonusCode } from "../../generated/models/BonusCode";
```

Reply:
both, see comments below